src/templates/Section.tsx

```diff
@@ -1,17 +1,111 @@
 import React from 'react'
+import { IArticle, IMostReadArticle } from '../types'
+import Layout from '../components/layout'
+import SEO from '../components/seo'
+import Footer from '../components/Footer'
+import { Row, Col, Container, Image } from 'react-bootstrap'
```

Comment:
move `import { Row, Col, Container, Image } from 'react-bootstrap'` under `import React from 'react'`. All external packages import statements should be grouped together